### DISCUSSION TITLE
人工智能將如何讓板球變得不可預測？

### DISCUSSION THREAD

**Discussion by titan92601** [1101-GS3073-C/Python-Community-GS3073-C#575] [科技新知]
這麼神奇的嗎？
我想都沒想過啊
https://www.analyticsinsight.net/how-will-artificial-intelligence-make-cricket-unpredictable/

**Comment by number32984** [comment]
你投變化球，我拿變化棒打變化球

**Comment by awehao** [comment]
拿顯卡打板球

**Comment by Raywei0214** [comment]
板球因何而發生？拉伯雷說過一句富有哲理的話，望見了海岸才溺死，是死得雙倍淒。這句話把我們帶到了一個新的維度去思考這個問題。

**Comment by larry40305** [comment]
現在各運動賽事也開始加入AI來分析賽事或用來輔助裁判
